test/manageMembers.spec.js

```diff
@@ -222,4 +222,84 @@ describe("manage members integration test", () => {
       expect(await ethersSafe.isOwner(charlie.address)).to.eq(true);
     });
   });
+
+  describe("when managing signers as safe", () => {
+    let { signers, memberToken, ethersSafe, createPodHelper, safeExecutionHelper } = {};
+    let [, , , charlie, dave] = [];
```

Comment:
what is this lol

Reply:
was just destructuring hack to make copy paste easier